modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -4105,9 +4105,16 @@ AUI.add(
 					var newFieldLocalizations = repeatedField.get(
 						'localizationMap'
 					);
-					var totalLocalizations = originalField.get(
-						'localizationMap'
-					);
+
+					var totalLocalizations;
+
+					if (originalField) {
```

Comment:
Hi @NemethNorbert,

In which use case the original field is `undefined`?

Reply:
Original field is undefined, when newNestedFields.length is higher then originalNestedFields.length. In this scenario, instance.populateBlankLocalizationMap will be called with an undefined originalNestedFields